template/Gemfile

```diff
@@ -3,3 +3,11 @@
 source 'https://rubygems.org'
 
 gemspec
+
+# This block allows you to dynamically install a different version of Faraday.
+# This is useful to test your middleware against multiple versions in the CI Matrix.
+# We suggest maintainers to support at least Faraday 1.x and 2.x, but that's up to you!
+# Set in `.github/workflows/ci.yaml`, can be safely removed if you don't need this feature.
+install_if -> { ENV['FARADAY_VERSION'] } do
```

Comment:
Rubocop had input about the ENV#[] method being used, and suggested a fetch with default (even when nil).

Fair enough. Perhaps we can use a local variable, to only fetch the value once from the ENV? This is very minor. But since this will run in many linting tools, it's good to be careful.

Reply:
Good shout, it doesn't cost me anything to make this a `fetch` 😄